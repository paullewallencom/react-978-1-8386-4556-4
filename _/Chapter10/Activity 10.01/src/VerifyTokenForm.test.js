import React from 'react';
import {render} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import VerifyTokenForm from './VerifyTokenForm';

test('renders correctly', () => {
  const {container} = render(<VerifyTokenForm/>);
  expect(container).toMatchSnapshot();
});
