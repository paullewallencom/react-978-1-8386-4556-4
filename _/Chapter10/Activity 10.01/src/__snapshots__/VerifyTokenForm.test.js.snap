// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <h1>
      Verify Token
    </h1>
    <form>
      <input
        name="token"
        type="text"
        value=""
      />
      <button
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
